### ISSUE TITLE
QuickLook plugin?

### ISSUE THREAD

**Issue by syberknight** [aonez/Keka#120]
does Keka have a Mac OS X QuickLook plugin in order to see what's inside a zip file from within the Finder?

**Comment by aonez** [comment]
It does not. It may be a great additional feature for the future.

**Comment by 3zero2** [comment]
I agree that this would be a great feature to have. Are there any plans to implement this please?

**Comment by ozguncagri** [comment]
Any updates about QuickLook plugin?

**Comment by gingerbeardman** [comment]
@ozguncagri as it says above, not until #100 is done

**Comment by ozguncagri** [comment]
@gingerbeardman ofc QL plugin obviously dependent on #100 I mean is there any updates in total (including any other dependent issues)? @aonez can you give us little brief about current status?

**Comment by aonez** [comment]
No news right now @ozguncagri

**Comment by Jax0rz** [comment]
I found a[ plugin](https://findergg.com) is very useful for all the quicklook needs.

**Comment by 3zero2** [comment]
Looks intersting indeed @Jax0rz

**Comment by gingerbeardman** [comment]
> I found a[ plugin](https://findergg.com) is very useful for all the quicklook needs.

"Requires macOS 10.15 or later."

But why?

**Comment by 3zero2** [comment]
> > I found a[ plugin](https://findergg.com) is very useful for all the quicklook needs.
> 
> "Requires macOS 10.15 or later."
> 
> But why?

It makes use of the QuickLook API which was introduced in 10.15. The previously used qlgenerator has been deprecated.

**Comment by gingerbeardman** [comment]
That's good to know! Thanks

**Comment by lnswlrd** [comment]
BetterZip has a freeware Quick Look generator/extension made to accompany their commercial product BetterZip. Note, you don't have to buy that software because Keka does it all. So just grab this free QL utility. 
https://macitbetter.com/BetterZip-Quick-Look-Generator/

**Comment by bitsnaps** [comment]
This app [Unzip - RAR ZIP 7Z Unarchiver](https://apps.apple.com/us/app/unzip-unarchive-expert/id1559340169) has that feature (both Mac & iOS).

**Comment by bigplayer-ai** [comment]
> BetterZip has a freeware Quick Look generator/extension made to accompany their commercial product BetterZip. Note, you don't have to buy that software because Keka does it all. So just grab this free QL utility. https://macitbetter.com/BetterZip-Quick-Look-Generator/

Quick Look generator/extension stopped working on macOS Sequoia 15.1, Apple has removed support for QuickLook generators and has moved to a different method in MacOS Sequoia (see: https://eclecticlight.co/2024/10/31/how-sequoia-has-changed-quicklook-and-its-thumbnails/). It only works currently if Keka Implemented this feature using the new method (only accessible via applications developers)

**Comment by bigplayer-ai** [comment]
> I found a[ plugin](https://findergg.com) is very useful for all the quicklook needs.

Quick Look generator/extension stopped working on macOS Sequoia 15.1, Apple has removed support for QuickLook generators and has moved to a different method in MacOS Sequoia (see: https://eclecticlight.co/2024/10/31/how-sequoia-has-changed-quicklook-and-its-thumbnails/). It only works currently if Keka Implemented this feature using the new method (only accessible via applications developers)
